src/studio-home/tabs-section/libraries-v2-tab/libraries-v2-filters/libraries-v2-order-filter-menu/index.jsx

```diff
@@ -0,0 +1,58 @@
+import { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import messages from './messages';
+
+import LibrariesV2FilterMenu from '../libraries-v2-filter-menu';
+
+const LibrariesV2OrderFilterMenu = ({ onItemMenuSelected, isFiltered }) => {
+  const intl = useIntl();
+
+  const libraryV2Orders = useMemo(
+    () => [
+      {
+        id: 'az-libraries-v2',
```

Comment:
Nit: I find this ID confusing. I think `sort-libraries-az` would be more clear.

Reply:
Good point, updated the Ids to make them more reader-friendly: https://github.com/openedx/frontend-app-course-authoring/pull/1117/commits/07542c01e975d0e52bce1ae3c50799f5ed4c9027